NomaiVR/Input/ControllerInput.cs

```diff
@@ -412,6 +413,13 @@ private static bool PostIsGamepadEnabled(bool __result)
                     return true;
                 }
 
+                private static bool PrefixGetAnyJoystickButtonPressed()
+                {
+                    //This method is only used in the intro screen and can sometimes give unexpected
+                    //phantom keys during startup sequence
+                    return false;
+                }
+
```

Comment:
Does this not prevent skipping the intro sequence? Also, if you just want to disable a method you can use and `Empty` patch instead of a `Prefix`.

Reply:
That method was checking for keys that are not actually used, so the intro scene is still skippable by pressing anything.
I'll use `Empty`, wasn't sure if it would break anything :P